javascript/reflexes.js

```diff
@@ -179,4 +203,10 @@ const setupDeclarativeReflexes = debounce(() => {
   emitEvent('stimulus-reflex:ready')
 }, 20)
 
-export { received, registerReflex, getReflexRoots, setupDeclarativeReflexes }
+export {
+  received,
+  getReflexElement,
+  getReflexOptions,
+  getReflexRoots,
+  scanForReflexes
+}
```

Comment:
I wonder if we should rename this file to something like `reflex_utils.js` to avoid any future confusion with the reflex store holding reflex instances. Not a blocker, just a thought as I'm scanning through the imports in various files on the PR.

Reply:
I think that what I should do is take your advice, but also split out the `received` callback into its own file; perhaps 'received.js` or `ingress.js`.

Reply:
Just reworked this topic in https://github.com/stimulusreflex/stimulus_reflex/pull/592/commits/8eca32e184291905c09ef8ca6e3ed9065c32a585 which moved the `received` function to a new file, `process.js` (name up for discussion!) while consolidating the various utils that had been in `reflexes.js` to `utils.js` where they belong.

Most significantly, this allowed me to rename `reflex_store.js` to the now available `reflexes.js` as nature intended. :+1: